modules/ROOT/pages/7.5-release-notes.adoc

```diff
@@ -193,7 +193,13 @@ This section describes issues that users of {productname} {release-version} may
 
 There <is one | are <number> known issue<s> in {productname} {release-version}.
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+== Known Issues
+
+=== Screenreader on safari has poor consistency of announcing aria-label
+// #TINY-11291 & #TINY-11430
+
+In Safari, when using a screen reader, the focus is incorrectly placed on the content of the input field instead of the input field itself. This behavior results in the screen reader not announcing the content of the `aria-label` associated with the input field.
```

Comment:
```suggestion
 In Safari, when using a screen reader on the color picker, the focus is mistakenly placed on the content within the input field rather than the field itself. This causes the screen reader to bypass announcing the aria-label associated with the field.
```